packages/sdk/package.json

```
@@ -0,0 +1,53 @@
+{
+  "dependencies": {
+    "@polkadot/api": "^16.4.1",
+    "stellar-sdk": "^13.3.0",
+    "viem": "^2.31.7"
+  },
+  "devDependencies": {
+    "@types/bun": "latest"
+  },
+  "exports": {
+    ".": {
+      "import": {
+        "browser": "./dist/browser/index.js",
+        "default": "./dist/browser/index.js",
+        "node": "./dist/node/index.js"
+      },
+      "require": {
+        "browser": "./dist/browser/index.cjs",
+        "default": "./dist/browser/index.cjs",
+        "node": "./dist/node/index.cjs"
+      },
+      "types": "./dist/index.d.ts"
+    },
+    "./package.json": "./package.json"
+  },
+  "files": [
+    "dist",
+    "README.md"
+  ],
+  "main": "./dist/browser/index.js",
+  "module": "./dist/browser/index.js",
+  "name": "@vortexfi/sdk",
+  "peerDependencies": {
+    "typescript": "^5"
+  },
+  "scripts": {
+    "build": "bun run clean && bun run build:types && bun run build:js",
+    "build:js": "bun run build:js:node && bun run build:js:browser",
+    "build:js:browser": "bun build ./src/index.ts --outdir ./dist/browser --target=browser --sourcemap=external",
+    "build:js:node": "bun build ./src/index.ts --outdir ./dist/node --target=node --sourcemap=external",
+    "build:types": "tsc -p tsconfig.json",
+    "clean": "rm -rf ./dist",
+    "dev": "bun run --watch ./src/index.ts",
+    "format": "prettier --write .",
+    "lint": "eslint . --ext .ts",
+    "prepublishOnly": "bun run build",
+    "test": "bun test",
+    "typecheck": "bun x --bun tsc"
+  },
+  "type": "module",
+  "types": "./dist/index.d.ts",
+  "version": "0.1.7"
```

Shall we set this to 
```suggestion
  "version": "1.0.0"
```